templates/preview-font.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang=en data-inscription={{self.inscription_id}}>
+  <head>
+    <meta charset=utf-8>
+    <link rel=stylesheet href=/static/preview-font.css></link>
+    <script src=/static/preview-font.js defer type=module></script>
```

Comment:
Can we do this without any JS, like this?

```suggestion
    <style>
      @font-face {
        font-family: 'Font';
        src: url(/content/{{self.inscription}}) format('{self.format}');
      }

      textarea {
        font-family: 'CustomFont';
        background-color: #292c2f;
        color: #98a3ad;
        border: 1px solid #292c2f;
        width: 90vw;
        height: 90vh;
        resize: none;
        padding: 10px;
        font-size: 24px;
    }

```

Reply:
Hmm yeah that seems cleaner. So would we do this logic serverside? 
```js
const contentType = response.headers.get("content-type");
let fontType = contentType.split("/")[1];
switch (fontType) {
    case "ttf":
        fontType = "truetype";
        break;
    case "otf":
        fontType = "opentype";
        break;
}
```
You had mentioned doing somthing serverside in the stream for the codepreview. Could you elaborate on how you would approach doing this? woff and woff2 the fontType can be the mime type however ttf and otf need this mapping.

Reply:
Yeah, I think it's better to avoid calculations in JS where possible. Our JS is untested, since frontend tests are slow and flakey, so having as little logic there as possible is best.

For the code preview, it would also be nice to pass the code type string instead of getting it from the content type, for the same reasons.